spec/requests/admin/api_controller_spec.rb

```diff
@@ -154,6 +200,16 @@
         expect(UserHistory.last.details).to eq(I18n.t("staff_action_logs.api_key.restored"))
       end
     end
+
+    describe '#scopes' do
+      it 'includes scopes' do
+        get '/admin/api/keys/scopes.json'
+
+        data = response.parsed_body
+
+        expect(data['scopes']).not_to be_empty
```

Comment:
I usually prefer to not use generic assertion like `not_to be_empty` because this test will pass even if `data['scopes']` contains random data.